alfresco-solrapi-client/alfresco-solrapi-client-api/src/main/java/eu/xenit/alfresco/client/solrapi/api/model/SolrNodeMetaData.java

```diff
@@ -2,10 +2,13 @@
 
 import java.util.List;
 import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Data;
 import lombok.Value;
 import lombok.experimental.Accessors;
 
-@Value
+@Data
```

Comment:
`@Data` implies setters - I would like to avoid that

Reply:
I understand

I will change that